### DISCUSSION TITLE
Display Customer and Job Notes in upcoming hires.

### DISCUSSION THREAD

**Discussion by Russell-Maint** [stayonhire/feedback#28] [Ideas / Wishlist]
Currently the "Upcoming hires" List on the mobile app shows the hire start date and all the assets due to start. This is great and our ground team use this feature everyday. 

Problem: 
Using this feature our ground team can't determine the specific customer the hire is for. This means we miss critical aspects of machine compliance.  Also, ground team can't see that job specific notes exist.
![image](https://github.com/stayonhire/feedback/assets/145405402/7c6e7d7c-0173-4a66-a73b-4e7d767a8f7d)

Possible Solution:
 Add two lines under hire date. 
Line 1 - Client
Line 2 - Job Notes
![image](https://github.com/stayonhire/feedback/assets/145405402/8904e9dc-d703-48d7-b811-91477154ca90)

**Comment by dylanhru** [comment]
Hi Rus,

While Tom and I have more grand plans for the mobile app (ie. all job functionality, notifications, to dos), we'll be deploying some small fixes that should try and remedy this use case in the short-term as follows:

1. A quick (and somewhat dirty label) will show the current and next job for each asset
![Screenshot 2023-10-03 at 11 29 13 am](https://github.com/stayonhire/feedback/assets/34927957/b755b699-7274-4296-b75f-bab5698ddb39)

2. A few 'next job' fields will be added to each asset's overview page
![Screenshot 2023-10-03 at 11 29 30 am](https://github.com/stayonhire/feedback/assets/34927957/c5da5ee7-271e-46d9-8b2a-11076dbb28ce)

These should be deployed today/tomorrow and will simply require a quick update/page refresh to display.

**Comment by Russell-Maint** [comment]
Thanks Dylan.

Can we include those labels on assets that are not yet on hire?

[image]
E.g 22-04,22-05 are heading to Glencore Rolleston. It would make it easier for the guys to know to check that specific site’s compliance status if it was displayed there.
Kind regards,

Russell Price | Maintenance Superintendent

M| 0423 774 856 E| ***@***.******@***.***> W|www.abpower.com.au<http://www.abpower.com.au/>

[image001.png]

On 3 Oct 2023, at 1:32 pm, Dylan Hrurukvit ***@***.***> wrote:

﻿

Hi Rus,

While Tom and I have more grand plans for the mobile app (ie. all job functionality, notifications, to dos), we'll be deploying some small fixes that should try and remedy this use case in the short-term as follows:

  1.  A quick (and somewhat dirty label) will show the current and next job for each asset
[Screenshot 2023-10-03 at 11 29 13 am]<https://user-images.githubusercontent.com/34927957/272139486-b755b699-7274-4296-b75f-bab5698ddb39.png>

  2.  A few 'next job' fields will be added to each asset's overview page
[Screenshot 2023-10-03 at 11 29 30 am]<https://user-images.githubusercontent.com/34927957/272139594-c5da5ee7-271e-46d9-8b2a-11076dbb28ce.png>

**Reply by dylanhru** [reply]
Yep, these labels will appear in all asset statuses. 

For example:

**Not yet on hire** - Will show 'Next' job tags
**Currently on hire** - Will show 'Current' (and 'Next' if available) tags

**Reply by dylanhru** [reply]
Still actively working on the 'Next' tags as we speak, though you will see these appear soon enough

**Comment by dylanhru** [comment]
These changes have now been deployed:
![Screenshot 2023-10-04 at 2 21 11 pm](https://github.com/stayonhire/feedback/assets/34927957/319a8468-ec3d-47e6-99c9-6c10575a6046)
![Screenshot 2023-10-04 at 2 20 55 pm](https://github.com/stayonhire/feedback/assets/34927957/bea8d814-96d4-4cad-8c6a-5c6f333f2bb3)
